themes/default/content/blog/mlc-guide/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Authoring Multi-Language Components"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-06-24T12:55:12-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test. 
+# Max length is 160 characters.
+meta_desc: A brief guide to authoring multi-language components
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+# meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - harry-liuson
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-interns
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+One of Pulumi's most powerful features is the component resource. A component resource is a reusable abstraction for cloud resources and behaviors (see [here](https://www.pulumi.com/docs/intro/concepts/resources/components/) for more details).
```

Comment:
```suggestion
One of Pulumi's most powerful features is the [component resource]({{< relref "/docs/intro/concepts/resources/components" >}}). A component resource is a reusable abstraction for cloud resources and behaviors.
```

We try our best to avoid here, click here, learn more links as they are not accessible, and make it difficult to know where they will take you if you happen to be scanning or navigating by links. Instead we strive to use links that are fully understandable outside of the context of the rest of the text.

We have a [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md), if ya have a few id check it out for a few more tips.